connect/client/mixins.py

```diff
@@ -60,7 +60,7 @@ def execute(self, method, path, **kwargs):
 
             if self.response.status_code == 204:
                 return None
-            if self.response.headers['Content-Type'] == 'application/json':
+            if 'application/json' in self.response.headers['Content-Type']:
```

Comment:
Based on our discussion, will be better to use startswith to check that condition.